Directory.Build.props

```diff
@@ -9,7 +9,7 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <NeutralLanguage>en-US</NeutralLanguage>
     <NoWarn>$(NoWarn);1591;1998;NU5105;CA1014;CA1508</NoWarn>
-    <DebugType>embedded</DebugType>
+    <DebugType>portable</DebugType>
```

Comment:
FWIW, this prevents clients from seeing debug symbols when debugging via NuGet package unless pdbs are included in the NuGet package and the client copies them into the build output -or- we figure out how to use symbol servers properly.

Reply:
I'm sure it's a bug in Rider. It seemed like it worked before but recently stopped working.